### PR TITLE
improve CONTRIBUTING.md

### PR DESCRIPTION
**Pull request by vieux**
This repo is a collection of sources, but doesn't accept direct contribution. Let's clarify this.

FYI @mavenugo @mataneja @tiborvass @mlaventure

### CODE REVIEW

File: CONTRIBUTING.md [hunk 1]
```diff
@@ -2,6 +2,123 @@
...
+applicable. This information will help us review and fix your issue faster.
+When sending lengthy log-files, consider posting them as a gist (https://gist.github.com).
+Don't forget to remove sensitive data from your logfiles before posting (you can
+replace those parts with "REDACTED").
+
+## Submitting pull requests
+
+Please see the corresponding `CONTRIBUTING.md` file of each component for more information:
+
 * Changes to the `engine` should be directed upstream to https://github.com/moby/moby
```
Comment (mataneja) [inline CONTRIBUTING.md]: after the moby evolution blog is posted, this should be updated to `moby/moby-core`

Comment (vieux) [inline CONTRIBUTING.md]: agreed.

File: CONTRIBUTING.md [hunk 2]
```diff
@@ -2,6 +2,123 @@
...
+* https://github.com/docker/for-win
+* https://github.com/docker/for-aws
+* https://github.com/docker/for-azure
+
+When reporting issues, always include:
+
+* The output of `docker version`.
+* The output of `docker info`.
+
+Also include the steps required to reproduce the problem if possible and
```
Comment (mlaventure) [inline CONTRIBUTING.md]: Maybe mention to also follow the issue template?

File: CONTRIBUTING.md [hunk 3]
```diff
@@ -2,6 +2,123 @@
...
 * Changes to the `packaging` should be directed upstream to https://github.com/docker/docker-ce-packaging
+
+## Docker community guidelines
+
+We want to keep the Docker community awesome, growing and collaborative. We need
+your help to keep it that way. To help with this we've come up with some general
+guidelines for the community as a whole:
+
+* Be nice: Be courteous, respectful and polite to fellow community members:
+  no regional, racial, gender, or other abuse will be tolerated. We like
```
Comment (andrewhsu) [inline CONTRIBUTING.md]: i'd suggest change for flow of words:

Be courteous, respectful, and polite to fellow community members. Regional, racial, gender, or other abuse ** will not** be tolerated.

Comment (vieux) [inline CONTRIBUTING.md]: 👍

File: CONTRIBUTING.md (hunk 2 above)
Comment (vieux) [inline CONTRIBUTING.md]: @mlaventure any suggestion for the wording ? :D

Comment (mlaventure) [inline CONTRIBUTING.md]: `If presented with a template when creating an issue, please follow its directions` ?

**Pull request merged**